### ISSUE TITLE
about the scores

### ISSUE THREAD

**Issue by Yang-L1** [hyangwinter/flownet3d_pytorch#5]
Hi, just a simple question. Did you train and evaluate your code? how is the endpoint errors compared to the office flownet3D implementation?

**Comment by hyangwinter** [comment]
The EPE is a litter higher than the office implementation in my experiment, about 0.24

**Comment by Yang-L1** [comment]
That's not a little higher. 0.16-->0.24 much higher. :) 
Do you have any idea what makes the difference?

**Comment by hyangwinter** [comment]
Sorry, I make a mistake, I confused training loss with EPE metric. I added the evaluation code and tested it again, here is my reault:
```
mean test loss: 0.022918        EPE 3D: 0.156013        ACC 3D: 0.193915        ACC 3D 2: 0.555054
```

**Comment by Yang-L1** [comment]
Good Job man! If the train\test splits are the same. 
Your implementation is even better than the numbers reported in the paper. 
epe 0.1694  acc1 25.37% acc2  57.85%
Do you mind uploading the trained model?

**Comment by hyangwinter** [comment]
I use the same data set and split as in the paper. The pretained model has been uploaded.

**Comment by Yang-L1** [comment]
Thanks, I obtained the same score with your model.

**Comment by Guptajakala** [comment]
> Sorry, I make a mistake, I confused training loss with EPE metric. I added the evaluation code and tested it again, here is my reault:
> 
> ```
> mean test loss: 0.022918        EPE 3D: 0.156013        ACC 3D: 0.193915        ACC 3D 2: 0.555054
> ```

@hyangwinter 
Hi, thanks for this great repo, I have starred it. Can you point me where is the above result printed? When I run main.py with --eval, I only get this.

```
==FINAL TEST==
mean test loss: 0.022395
FINISH
```

**Comment by munib94** [comment]
> > Sorry, I make a mistake, I confused training loss with EPE metric. I added the evaluation code and tested it again, here is my reault:
> > ```
> > mean test loss: 0.022918        EPE 3D: 0.156013        ACC 3D: 0.193915        ACC 3D 2: 0.555054
> > ```
> 
> @hyangwinter
> Hi, thanks for this great repo, I have starred it. Can you point me where is the above result printed? When I run main.py with --eval, I only get this.
> 
> ```
> ==FINAL TEST==
> mean test loss: 0.022395
> FINISH
> ```

I would also like to know how you printed out those metrics. Thank you.

**Comment by hyangwinter** [comment]
I had updated the code in main.py for the evaluation of EPE metric.

**Comment by munib94** [comment]
Thank you for including the metrics code! However, when I trained from scratch and evaluated, I got the following results:
```
==FINAL TEST==
mean test loss: 0.024785	EPE 3D: 0.164090	ACC 3D: 0.185281	ACC 3D 2: 0.528741
```
While it is different from what you got, the EPE 3D is still better than the paper:
EPE: 0.1694 ACC 1: 25.37% ACC 2: 57.85%
